types/activex-diskquota/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
For new packages, this file should only `"extends": "dtslint/dt.json",`

Reply:
Pending an answer [here](https://github.com/Microsoft/TypeScript/issues/20703#issuecomment-362272300).

Reply:
Even you want to disable the 2 rules, you can do it by adding `// tslint:disable interface-name` in source code